### PR TITLE
Add support for region

### PR DESCRIPTION
**Pull request by brtkwr**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -138,6 +138,7 @@
         cacert: "{{ os_images_cacert | default(omit) }}"
         interface: "{{ os_images_interface | default(omit, true) }}"
```
Comment (markgoddard) [inline tasks/main.yml]: nit: connection/auth parameters are grouped first

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 1 above)
Comment (brtkwr) [reply tasks/main.yml]: Done

File: tasks/main.yml [hunk 2]
```diff
@@ -137,6 +137,7 @@
         auth: "{{ os_images_auth }}"
         cacert: "{{ os_images_cacert | default(omit) }}"
         interface: "{{ os_images_interface | default(omit, true) }}"
+        region: "{{ os_images_region | default(omit) }}"
```
Comment (markgoddard) [inline tasks/main.yml]: Docs say it's region_name: https://docs.ansible.com/ansible/2.9/modules/os_image_module.html

File: defaults/main.yml [hunk 3]
```diff
@@ -62,6 +62,9 @@ os_images_auth:
 # public, admin, or internal.
 os_images_interface:
 
+# Default region for OpenStack endpoint.
+os_images_region:
```
Comment (markgoddard) [inline defaults/main.yml:77]: os_images_region_name if we change the module args

**Review by oneswig — APPROVED** [review]
Fixes applied and tested with and without the parameter set

**Review by markgoddard — APPROVED** [review]

**Pull request merged**